test/test_linalg.py

```diff
@@ -3991,13 +3992,19 @@ def _gen_shape_inputs_linalg_triangular_solve(self, shape, dtype, device, well_c
                 size_b = size_b[1:]
 
             if well_conditioned:
-                PLU = torch.linalg.lu(make_randn(*size_a))
+                X = make_randn(*size_a)
+                X = X + X.mH
+                U = torch.linalg.svd(X)[0]
```

Comment:
Alternatively, one could use QR, but I decided to stick to SVD for probably a bit easier understanding...
But I can change that to improve efficiency if needed...